i18n/en.json

```diff
@@ -309,7 +313,7 @@
     "transcoding_reference_frames_description": "The number of frames to reference when compressing a given frame. Higher values improve compression efficiency, but slow down encoding. 0 sets this value automatically.",
     "transcoding_required_description": "Only videos not in an accepted format",
     "transcoding_settings": "Video Transcoding Settings",
-    "transcoding_settings_description": "Manage the resolution and encoding information of the video files",
+    "transcoding_settings_description": "Manage which videos to transcode and how to encode them",
```

Comment:
"process" or "transcode" is more accurate than "encode".

Reply:
Or "re-encode" I suppose. But yeah, I like "process".